File: python/leetcode283.py
class Solution:
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        ls = len(nums)
        i = 0
        while i < ls:
            if nums[i] == 0:
                temp = nums.pop(i)
                nums.append(temp)
                ls-=1

File: leetcodec++/leetcode283.cpp
//
// Created by 黄敏辉 on 2019-09-18.
//

class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int len = nums.size();
        for (int i = 0; i < len; i ++){
            if (nums[i] == 0){
                nums.erase(nums.begin() + i);
                nums.push_back(0);
                len--;
                i--;
            }
        }
    }
};